plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/github.ts

```diff
@@ -99,6 +100,14 @@ export function createPublishGithubAction(options: {
               },
             },
           },
+          topics: {
+            title: 'Topics',
+            description: 'Uppercase letters no allowed',
```

Comment:
Wouldn't it be better to lowercase all topics in the action instead of relying on the user to do the right thing? :)

Reply:
We could do that yes. I was a bit unsure whether that would be odd as a user, but it might be better than nothing.

Reply:
I have added lowercasing 👍